presto-main/src/main/java/io/prestosql/server/QueryProgressStats.java

```diff
@@ -68,8 +68,8 @@ public static QueryProgressStats createQueryProgressStats(BasicQueryStats queryS
                 queryStats.getTotalScheduledTime().toMillis(),
                 queryStats.getUserMemoryReservation().toBytes(),
                 queryStats.getPeakUserMemoryReservation().toBytes(),
-                queryStats.getRawInputPositions(),
-                queryStats.getRawInputDataSize().toBytes(),
+                queryStats.getPhysicalInputPositions() + queryStats.getInternalNetworkInputPositions(),
```

Comment:
I don't think `queryStats.getPhysicalInputPositions() + queryStats.getInternalNetworkInputPositions()` is equivalent to `rawInputPositions`. `QueryStateMachine` has:
```
            if (plan != null && plan.getPartitionedSourceNodes().stream().anyMatch(TableScanNode.class::isInstance)) {
                rawInputDataSize += stageStats.getRawInputDataSize().toBytes();
                rawInputPositions += stageStats.getRawInputPositions();

                processedInputDataSize += stageStats.getProcessedInputDataSize().toBytes();
                processedInputPositions += stageStats.getProcessedInputPositions();
            }
```
Addition here is conditional. It seems that `processedInputPositions` is equivalent to `rawInputPositions`. However, there is not equivalence for `rawInputDataSize`.